torch/fx/passes/runtime_assert.py

```diff
@@ -194,6 +206,14 @@ def go(node, keypath):
                             and isinstance(keypath[0], CallMethodKey)
                             and isinstance(keypath[1], pytree.SequenceKey)
                         ):
+                            if keypath[0].name == "size":
```

Comment:
For completeness, you should do `stride` and (not here, but below) `storage_offset` as well